### PR TITLE
Bump version to 1.2.0 and update CHANGELOG

### PR DESCRIPTION
**Pull request by systemcatch**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪  Added **tests** for changed code.
- [ ] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [ ] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚  Updated **documentation** for changed code.
- [ ] ⏩  Code is **up-to-date** with the `master` branch.

WIP, need to provide the full list of locales that now support dehumanize.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+1.2.0 (2021-09-06)
+------------------
+
+- [NEW] Added Albanian, Tamil and Zulu locales.
+- [NEW] Added support for ``Decimal`` as input to ``arrow.get()``.
+- [FIX] The Arabic, Czech, Finnish, Hebrew, Icelandic, Slavic, Slovak and Thai locales now support ``dehumanize``.
```
Comment (anishnya) [inline CHANGELOG.rst:9]: Finnish, Nepali, Estonian and Zulu are the only new locales that support dehumanize at the moment

**Review by anishnya — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (systemcatch) [reply CHANGELOG.rst:9]: Ahh ok so the others have been refactored in preparation for supporting it?

Comment (anishnya) [reply CHANGELOG.rst:9]: Correct.

**Review by anishnya — APPROVED** [review]
LGTM!

**Comment by cyriaka90** [comment]
Hey! I was wondering why there's no tag on GitHub and no release on PyPI. Is this on purpose? Was looking forward to using the new version ;)

**Pull request merged**